week3/homework/src/actions/readTodo.js

```diff
@@ -0,0 +1,17 @@
+'use strict';
+
+function readTodo(todo, request, response) {
+  const id = request.params.id;
+  todo
+    .readTodo(id)
```

Comment:
is it guaranteed that you will always find a todo with a given id?

Reply:
If i understand your question correctly, my answer is yes. 
Without a correct id you will get an empty object as a result.